velox/substrait/SubstraitToVeloxExpr.h

```diff
@@ -63,6 +64,10 @@ class SubstraitVeloxExprConverter {
       const RowTypePtr& inputType);
 
  private:
+  /// The entry of constructing ArrayVector from list literal
```

Comment:
Comments should end with a period. Please, fix throughout.

This comment is not particularly informative and "The entry of" is specifically not necessary. Perhaps, shorten to "Convert list literal to ArrayVector."

Reply:
@zhejiangxiaomai Maybe this comment got missed as I don't see the code updated. Copying the comment.

----
Comments should end with a period. Please, fix throughout.

This comment is not particularly informative and "The entry of" is specifically not necessary. Perhaps, shorten to "Convert list literal to ArrayVector."